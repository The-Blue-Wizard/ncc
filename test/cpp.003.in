/* make sure #undef works properly */

#define x 32
x
#undef x
#define x 64
#define y 128
x y
#undef x
#define x(a) #a
x(test)
