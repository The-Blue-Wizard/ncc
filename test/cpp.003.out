# 1 "cpp.003.in"



32 



64 128 


"test" 
